tests/rust/roundtrips/pinhole/src/main.rs

```diff
@@ -0,0 +1,36 @@
+//! Logs a `Pinhole` archetype for roundtrip checks.
+
+use rerun::{archetypes::Pinhole, datatypes::Mat3x3, external::re_log, RecordingStream};
+
+#[derive(Debug, clap::Parser)]
+#[clap(author, version, about)]
+struct Args {
+    #[command(flatten)]
+    rerun: rerun::clap::RerunArgs,
+}
+
+fn run(rec: &RecordingStream, _args: &Args) -> anyhow::Result<()> {
+    rec.log(
+        "pinhole",
+        &Pinhole::new(Mat3x3::from([[3., 0., 1.5], [0., 3., 1.5], [0., 0., 1.]]))
```

Comment:
This is row-major layout... I'm surprised that this passes the round-trip. I would have expected this to be specified column-major to match the python round-trip.

Reply:
huh, yeah now that you point it out that's odd. I might have messed up Mat3x3 serialization somwhere?

Reply:
good news, this *does* fail. I'm fairly sure I had this test passing when I wrote it which means I fixed it later on - my *first* implementation of matrix3 implementation was wrong, but then I got it right while chipping away on this PR 😄